controllers/admin/AdminMetaController.php

```diff
@@ -746,7 +746,9 @@ public function getRobotsContent()
         );
 
         // Directories
-        $tab['Directories'] = array('classes/', 'config/', 'download/', 'mails/', 'modules/', 'translations/', 'tools/');
+        $tab['Directories'] = array('cache/', 'classes/', 'config/', 'controllers/',
```

Comment:
After comparing with 1.6, /modules/ was put in the file because on top, we have :
`Allow: */modules/*.css`
`Allow: */modules/*.js`
`Allow: */modules/*.png`
`Allow: */modules/*.jpg`
**so we can keep modules ?**